### PR TITLE
Create test snap for ethereum-provider endowment

### PR DESCRIPTION
**Pull request by bowensanders**
This creates a test snap for the Ethereum-provider endowment.
See #165

### CODE REVIEW

File: packages/ethprovider/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# JSON-RPC Permissions Test Snap
```
Comment (FrederikBolding) [inline packages/ethprovider/README.md:1]: This README doesn't match the snap

File: packages/ethprovider/snap.manifest.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "version": "5.3.0",
+  "description": "An test Snap for ethereum-provider endowment.",
```
[the remark this answers is not in the document]
Comment (bowensanders) [reply packages/ethprovider/snap.manifest.json]: fixed in latest commit

File: packages/ethprovider/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { OnRpcRequestHandler } from '@metamask/snaps-types';
+
+// eslint-disable-next-line jsdoc/require-jsdoc
```
[the remark this answers is not in the document]
Comment (bowensanders) [reply packages/ethprovider/src/index.ts]: fixed in latest commit

File: packages/ethprovider/README.md (hunk 1 above)
Comment (bowensanders) [reply packages/ethprovider/README.md]: fixed in latest commit

**Review by david0xd — APPROVED** [review]

**Pull request merged**